digital_asset_types/src/dao/generated/backfill_items.rs

```diff
@@ -20,6 +20,7 @@ pub struct Model {
     pub slot: i64,
     pub force_chk: Option<bool>,
     pub backfilled: Option<bool>,
+    pub failed: Option<bool>,
```

Comment:
Just wondering, why is this (and maybe the others as well) an `Option<bool>` instead of just a `bool`?

Reply:
The answer is that SeaORM uses Option when the database field can also be NULL.

However, this question made me think that I don't need to allow NULL for any of the columns in `backfill_items`  so I updated the table and generated code in my latest commit.

Reply:
The answer is that SeaORM uses Option when the database field can also be NULL.

However, this question made me think that I don't need to allow NULL for any of the columns in `backfill_items`  so I updated the table and generated code in my latest commit.